docs/resources/tutorials/core/qos/mutable/vulcanexus_change_mutable_qos/src/change_mutable_qos_publisher.cpp

```diff
@@ -0,0 +1,111 @@
+// Copyright 2016 Proyectos y Sistemas de Mantenimiento SL (eProsima).
```

Comment:
Have we run ament_uncrustify on these cpp files? I think we should

Reply:
Done